dependencies.yaml

```diff
@@ -388,7 +388,7 @@ dependencies:
               - *libcusparse114
   cuda_wheels:
     specific:
-      - output_types: pyproject
+      - output_types: [requirements, pyproject]
```

Comment:
This should be safe: https://github.com/rapidsai/raft/pull/2563#discussion_r1938056656